src/Mod/Arch/ArchComponent.py

```diff
@@ -741,22 +1050,56 @@ def isStandardCase(self,obj):
 
 
 class ViewProviderComponent:
+    """A default View Provider for Component objects.
 
-    "A default View Provider for Component objects"
+    Acts as a base for all other Arch view providers. It's properties and
```

Comment:
No need for apostrophe.

Reply:
Resolved in 0feb9b8ab268ecca9352d4f598f12ab066ff207e